.github/CODEOWNERS

```diff
@@ -31,3 +31,21 @@ Dockerfile @crossedfall
 
 /code/game/gamemodes/dynamic @itsmeow
 /code/modules/modular_computers @itsmeow
+/tools/MapTileAggregator.py @itsmeow
+/code/modules/admin/verbs/_help.dm @itsmeow
+/code/modules/admin/verbs/adminhelp.dm @itsmeow
+/code/modules/admin/verbs/mentorhelp.dm @itsmeow
+/code/modules/admin/player_panel.dm @itsmeow
+/code/modules/antagonists/fugitive/ @itsmeow
```

Comment:
```suggestion
```
I don't think you need to own fugitives. It's a ported game mode.

Reply:
I also rewrote almost all the fugitives code

Reply:
Personally I don't think its complex / enough of a framework system to deem it super necessary, you can't really mess up edits to fugative code and everyone can review it equally as well.